### PR TITLE
chore: Add schedule for production deploys

### PR DESCRIPTION
**Pull request by joshlarson**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207005109039605

### CODE REVIEW

File: .github/workflows/deploy-prod.yml [hunk 1]
```diff
@@ -2,6 +2,10 @@ name: Deploy to Prod
 
 on:
   workflow_dispatch:
+  schedule:
+    # Runs every Monday and Thursday at 15:39 UTC, which is either
+    # 10:49 AM or 11:49 AM ET, depending on Daylight Savings Time
+    - cron: "49 15 * * 1,4"
```
[the remark this answers is not in the document]
Comment (firestack) [reply .github/workflows/deploy-prod.yml:10]: [x-posting from slack]
I like this reasoning and this makes sense to me.

As for the time of day, if we had this run between 0900-1000 then we'd know about it before any meetings that day.

But, 0900-1000 is closer to peak hours so it's maybe not the best time for us to deploy something when we don't know how smoothly it goes yet.

---

I'm fine with leaving it as is though 👍🏻

File: .github/workflows/deploy-ecs.yml [hunk 2]
```diff
@@ -4,6 +4,12 @@ on:
   workflow_dispatch:
   push:
     branches: ["main"]
+  schedule:
+    # Runs every Monday and Thursday at 15:39 UTC, which is either
+    # 8:49 AM or 9:49 AM ET, depending on Daylight Savings Time. This
+    # is two hours before the prod deploy, so we can see how dev is
+    # doing before approving a prod deploy.
+    - cron: "49 13 * * 1,4"
```
Comment (firestack) [inline .github/workflows/deploy-ecs.yml:12]: > https://github.com/mbta/ors-deploy/pull/9/files#r1570895037

Oh actually, what if we have this run before the `prod` deploy runs so we potentially get a heads up before the `prod` deploy runs?

File: .github/workflows/deploy-prod.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (firestack) [reply .github/workflows/deploy-prod.yml:10]: Oh actually, I like this time for prod, but [what if we run this earlier for dev deploys](https://github.com/mbta/ors-deploy/pull/9#discussion_r1570896624)?

**Review by firestack — APPROVED** [review]

File: .github/workflows/deploy-ecs.yml (hunk 2 above)
Comment (firestack) [reply .github/workflows/deploy-ecs.yml:12]: Whoops, I uhh, just stated what it was already doing lol 😅

Comment (joshlarson) [reply .github/workflows/deploy-ecs.yml:12]: I guess that confirms it's a good idea!

**Pull request merged**